plugins/modules/zos_backup_restore.py

```diff
@@ -732,28 +732,7 @@ def hlq_type(contents, dependencies):
         raise ValueError("hlq_type is only valid when operation=restore.")
     if not match(r"^(?:[A-Z$#@]{1}[A-Z0-9$#@-]{0,7})$", contents, IGNORECASE):
         raise ValueError("Invalid argument {0} for hlq_type.".format(contents))
-    return contents.upper()
-
-
-def hlq_default(contents, dependencies):
-    """Sets the default HLQ to use if none is provided.
-
-    Parameters
-    ----------
-    contents : str
-        The HLQ to use.
-    dependencies : dict
-        Any dependent arguments.
-
-    Returns
-    -------
-    str
-        The HLQ to use.
-    """
-    hlq = None
-    if dependencies.get("operation") == "restore":
-        hlq = datasets.get_hlq()
-    return hlq
+    return contents
```

Comment:
i'm curious why the HLQ no longer needs to be upper-cased

Reply:
We already check [lines before](https://github.com/ansible-collections/ibm_zos_core/blob/10ac10cd7d08ec6cbcde8380ff501f1274ed6b52/plugins/modules/zos_backup_restore.py#L733)  if the HLQ is in upper case if not we already tell the user that is invalid